### PR TITLE
Prevent NPE when calling `ofInstant`. Fixes #20

### LINKED ISSUE #20: NPE on method `Row.zonedDateTimeOrNull`
https://github.com/seratch/kotliquery/blob/d234101c38723374e85f9628549def40842b6f66/src/main/kotlin/kotliquery/Row.kt#L245

The first argument to `ZonedDateTime.ofInstant` mustn't be null:
```kotlin
    public static ZonedDateTime ofInstant(Instant var0, ZoneId var1) {
        Objects.requireNonNull(var0, "instant");
```

### PR DESCRIPTION
**Pull request by elifarley**
This change is [Reviewable](https://reviewable.io/reviews/seratch/kotliquery/21)

### CODE REVIEW

File: src/main/kotlin/kotliquery/Row.kt [hunk 1]
```diff
@@ -241,30 +241,38 @@ data class Row(
     fun zonedDateTime(columnIndex: Int): ZonedDateTime {
         return zonedDateTimeOrNull(columnIndex)!!
     }
-    fun zonedDateTimeOrNull(columnIndex: Int): ZonedDateTime? {
-        return nullable(ZonedDateTime.ofInstant(sqlTimestampOrNull(columnIndex)?.toInstant(), ZoneId.systemDefault()))
-    }
+    fun zonedDateTimeOrNull(columnIndex: Int) = nullable(
```
Comment (seratch) [inline src/main/kotlin/kotliquery/Row.kt]: Please don't remove the declaration of return types here.

**Comment by seratch** [comment]
The changes themselves look fine to me.

File: src/main/kotlin/kotliquery/Row.kt (hunk 1 above)
Comment (elifarley) [reply src/main/kotlin/kotliquery/Row.kt]: Return types are back :)

**Comment by elifarley** [comment]
@seratch I've put type declarations back :)

**Review by seratch — APPROVED** [review]

**Pull request merged**